clab/general.go

```diff
@@ -0,0 +1,130 @@
+package clab
```

Comment:
let's rename this file to `clab/hosts.go` maybe? `general.go` doesn't say anything about the purpose of it